core/shared/src/main/scala/org/http4s/Entity.scala

```diff
@@ -31,7 +31,12 @@ sealed trait Entity[+F[_]] {
 }
 
 object Entity {
-  def apply[F[_]](body: EntityBody[F], length: Option[Long] = None): Entity[F] =
+
+//  @deprecated("Use Entity.stream instead", "1.0.0-M32")
+//  def apply[F[_]](body: Stream[F, Byte], length: Option[Long] = None): Entity[F] =
+//    stream(body, length)
```

Comment:
I would keep the deprecation to give people a nudge in the right direction.  I don't see a lot of Entity creation directly in user code, but still, we need to consider the experience of people moving to 1.0.

Reply:
I was commenting it in order to reveal compile errors... kept.